server/src/v2/middlewares/verify.js

```diff
@@ -0,0 +1,82 @@
+import jwt from 'jsonwebtoken';
+import pool from '../models/configDB';
+
+export default class Middleware {
+  static async verifyNewUser(req, res, next) {
+    const { email } = req.body;
+    try {
+      const { rows } = await pool.query('SELECT * FROM users WHERE email = $1', [email]);
+      if (rows[0]) {
+        res.status(400).json({
+          status: 'error',
+          msg: 'Your email is already registered in the app, you are only allowed to have one account.',
+        });
+      }
+    } catch (err) {
+      res.status(500).json({ msg: err });
+    }
+    next();
+  }
+
+  /* static async verifyExistingUser(req, res, next) {
```

Comment:
@JackieBinya is this commented code is not being used; remove it. Otherwise, uncomment it and allow it to work with the current refactoring.